README.md
BandharPhirus.com Source Code
-----------------------------
Michael Butar butar
indigomike7@gmail.com

Laravel + mysql + bootstrap html

cometchat3/CHANGELOG.txt
------------------------------------------------------------------------------------
 CometChat Change Log
------------------------------------------------------------------------------------

------------------------------------------------------------------------------------
CometChat v7.0.2 (2018-02-09)
------------------------------------------------------------------------------------

+ Fixes

  + Web
  - Fixed issue while logging out in Embedded Layout
  - Fixed issue with multiple message alerts during Kick and Ban in group
  - Fixed UI and CSS issues in Docked Layout
  - Miscellaneous Bug fixes

------------------------------------------------------------------------------------
CometChat v7.0.1 (2018-02-06)
------------------------------------------------------------------------------------

+ Fixes

  + Web
  - Fixed issue with Joomla CMS plugin
  - Fixed issue with friends functionality for custom integrations
  - Fixed issue with multiple message alerts when deleting a group
  - Fixed issue with mooSocial Integration for CometChat Cloud

+ Features

  + Web
  - Added API to list users and their details for CometChat Cloud
  - Added option to show offline users in Contacts list for CometChat Cloud

------------------------------------------------------------------------------------
CometChat v7.0.0 (2018-01-30)
------------------------------------------------------------------------------------

+ Fixes

  + Web
  - Fixed issue with Guest login in Social Authentication
  - Fixed issue while logging in mobile devices for Social Authentication
  - Miscellaneous Bug fixes

+ Features

  + Web
  - Added Role Based Access Control for Joomla CMS

------------------------------------------------------------------------------------
CometChat v6.9.29 (2018-01-23)
------------------------------------------------------------------------------------

+ Fixes

  + Web
  - Added support for Voice & Video Chat in modern mobile browsers (beta)
  - Changed the appearance for Voice & Video Chat
  - Fixed issue with Jomsocial plugins
  - Fixed issue with Sharetribe Integration for CometChat Cloud
  - Miscellaneous Bug fixes

+ Enhancements

  + Web
  - Added setting to disable Docked Layout in CometChat Admin Panel

------------------------------------------------------------------------------------
CometChat v6.9.28 (2018-01-17)
------------------------------------------------------------------------------------

+ Fixes

  + Web
  - Fixed issue with Graph in CometChat Admin Panel Dashboard
  - Fixed issue while searching detailed logs in CometChat Admin Panel
  - Miscellaneous Bug fixes

+ Enhancements

  + Web
  - Added OpenCart Integration

------------------------------------------------------------------------------------
CometChat v6.9.27 (2018-01-16)
------------------------------------------------------------------------------------

+ Fixes

  + Web
  - Fixed issue with sound notification in Groups for Embedded Layout
  - Miscellaneous Bug fixes

+ Enhancements

  + Web
  - Added Graphical Representation in CometChat Admin Panel Dashboard

------------------------------------------------------------------------------------
CometChat v6.9.26 (2018-01-12)
------------------------------------------------------------------------------------

+ Fixes

  + Web
  - Fixed issue while Receiving unread messages
  - Fixed issue with Audio/Video Chat in Group
  - Fixed issue with Guest stats count in CometChat Admin Panel Dashboard
  - Miscellaneous Bug fixes

+ Enhancements

  + Web
  - Added New updated WordPress Plugin

------------------------------------------------------------------------------------
CometChat v6.9.25 (2018-01-09)
------------------------------------------------------------------------------------

+ Fixes

  + Web
  - Fixed issue with stats count in CometChat Admin Panel Dashboard
  - Miscellaneous Bug fixes

+ Enhancements

  + Web
  - Added Guest login feature with Social Authentication

------------------------------------------------------------------------------------
CometChat v6.9.24 (2018-01-02)
------------------------------------------------------------------------------------

+ Fixes

  + Web
  - Fixed issue while displaying stickers in Real Time Monitor Window
  - Miscellaneous Bug fixes

+ Enhancements

  + Web

cometchat3/lang.php
<?php

///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

/* LANGUAGE */
$rtl									=	setLanguageValue('rtl','0',$lang,'core','default');
$language['title'] 						=	setLanguageValue('title','Chat',$lang,'core','default');
$language['chat_options']				=	setLanguageValue('chat_options','Chat Options',$lang,'core','default');
$language['type_status']				=	setLanguageValue('type_status','Type your status and hit the enter key!',$lang,'core','default');
$language['status_message']				=	setLanguageValue('status_message','Status Message',$lang,'core','default');
$language['status_option_a']			=	setLanguageValue('status_option_a','Available',$lang,'core','default');
$language['status_option_b']			=	setLanguageValue('status_option_b','Busy',$lang,'core','default');
$language['status_option_i']			=	setLanguageValue('status_option_i','Invisible',$lang,'core','default');
$language['blank1']						=	setLanguageValue('blank1','',$lang,'core','default');
$language['blank2']						=	setLanguageValue('blank2','',$lang,'core','default');
$language['loggedout_tooltip']			=	setLanguageValue('loggedout_tooltip','Please login to use chat',$lang,'core','default');
$language['chat_now_tab']				=	setLanguageValue('chat_now_tab','Chats',$lang,'core','default');
$language['me']							=	setLanguageValue('me','Me',$lang,'core','default');
$language['status_option_o']			=	setLanguageValue('status_option_o','Offline',$lang,'core','default');
$language['who_is_online']				=	setLanguageValue('who_is_online','Who\'s Online',$lang,'core','default');
$language['sound_notifications']		=	setLanguageValue('sound_notifications','Sound Notifications',$lang,'core','default');
$language['no_friends']					=	setLanguageValue('no_friends','You have no friends in your friend list, please add a few friends to use chat',$lang,'core','default');
$language['new_messages_notify']		=	setLanguageValue('new_messages_notify','New Messages...',$lang,'core','default');
$language['blank3']						=	setLanguageValue('blank3','',$lang,'core','default');
$language['user_tab_o']					=	setLanguageValue('user_tab_o','Offline',$lang,'core','default');
$language['find_a_user']				=	setLanguageValue('find_a_user','Search',$lang,'core','default');
$language['user_online']				=	setLanguageValue('user_online','is online',$lang,'core','default');
$language['user_offline']				=	setLanguageValue('user_offline','is offline',$lang,'core','default');
$language['user_busy']					=	setLanguageValue('user_busy','is online(busy)',$lang,'core','default');
$language['set_my_status']				=	setLanguageValue('set_my_status','Set my status',$lang,'core','default');
$language['online_status']				=	setLanguageValue('online_status','Online Status',$lang,'core','default');
$language['popup_notifications']		=	setLanguageValue('popup_notifications','Popup Notifications',$lang,'core','default');
$language['blank4']						=	setLanguageValue('blank4','',$lang,'core','default');
$language['start_chatting']				=	setLanguageValue('start_chatting','Start chatting!',$lang,'core','default');
$language['close']						=	setLanguageValue('close','Close',$lang,'core','default');
$language['no_users_online']			=	setLanguageValue('no_users_online','No contacts available',$lang,'core','default');
$language['no_friends_online']			=	setLanguageValue('no_friends_online','No contacts available',$lang,'core','default');
$language['status_available']					=	setLanguageValue('status_a','I\'m available',$lang,'core','default');
$language['status_busy']					=	setLanguageValue('status_b','I\'m busy',$lang,'core','default');
$language['status_offline']					=	setLanguageValue('status_o','I\'m offline',$lang,'core','default');
$language['status_invisible']					=	setLanguageValue('status_i','I\'m offline',$lang,'core','default');
$language['status_away']					=	setLanguageValue('status_aw','I\'m away',$lang,'core','default');
$language['separator']					=	setLanguageValue('separator',' | ',$lang,'core','default');
$language['and']						=	setLanguageValue('and','and ',$lang,'core','default');
$language['or']						=	setLanguageValue('or','or ',$lang,'core','default');
$language['more_notifications']			=	setLanguageValue('more_notifications',' more notifications',$lang,'core','default');
$language['close_existing_popup']		=	setLanguageValue('close_existing_popup','This popup is already open. Please close the existing popup and try again.',$lang,'core','default');
$language['navigate_page']				=	setLanguageValue('navigate_page','Navigating away from this page will close your chat popups.',$lang,'core','default');
$language['others']						=	setLanguageValue('others','Others',$lang,'core','default');
$language['loading']					=	setLanguageValue('loading','Loading...',$lang,'core','default');
$language['blank5']						=	setLanguageValue('blank5','',$lang,'core','default');
$language['my_name']					=	setLanguageValue('my_name','Name',$lang,'core','default');
$language['set_my_name']				=	setLanguageValue('set_my_name','Set my name',$lang,'core','default');
$language['type_name_status']			=	setLanguageValue('type_name_status','Type your Name/Status and hit the enter key!',$lang,'core','default');
$language['messages_from']				=	setLanguageValue('messages_from','Messages from',$lang,'core','default');
$language['view']						=	setLanguageValue('view','Click here to view',$lang,'core','default');
$language['new_message_from']			=	setLanguageValue('new_message_from','New Message from',$lang,'core','default');
$language['updated']					=	setLanguageValue('updated','Updated',$lang,'core','default');
$language['failed']						=	setLanguageValue('failed','Failed',$lang,'core','default');
$language['chat_now']					=	setLanguageValue('chat_now','Chat Now',$lang,'core','default');
$language['read_message']				=	setLanguageValue('read_message','You have received a new chat message. Click OK to read',$lang,'core','default');
$language['spam']						=	setLanguageValue('spam','Please don\'t spam your friend',$lang,'core','default');
$language['new_messages']				=	setLanguageValue('new_messages','New messages',$lang,'core','default');
$language['send_a_message']				=	setLanguageValue('send_a_message','Send a message...',$lang,'core','default');
$language['loggedout']					=	setLanguageValue('loggedout','You have logged out',$lang,'core','default');
$language['no_change']					=	setLanguageValue('no_change','No change',$lang,'core','default');
$language['no_user_found']				=	setLanguageValue('no_user_found','No user found',$lang,'core','default');
$language['message_with_smiley']		=	setLanguageValue('message_with_smiley','Has sent you a message with smiley.',$lang,'core','default');
$language['web']						=	setLanguageValue('web','Web',$lang,'core','default');
$language['mobile']						=	setLanguageValue('mobile','Mobile',$lang,'core','default');
$language['minimize_user_tab']			=	setLanguageValue('minimize_user_tab','Minimize User Tab',$lang,'core','default');
$language['minimize_setting_tab']		=	setLanguageValue('minimize_setting_tab','Close Options Tab',$lang,'core','default');
$language['clear_conversation']			=	setLanguageValue('clear_conversation','Clear Conversation',$lang,'core','default');
$language['report_conversation']		=	setLanguageValue('report_conversation','Report Conversation',$lang,'core','default');
$language['share_image']				=	setLanguageValue('share_image','Share Image',$lang,'core','default');
$language['capture_photo']				=	setLanguageValue('capture_photo','Capture Photo',$lang,'core','default');
$language['share_video']				=	setLanguageValue('share_video','Share Video',$lang,'core','default');
$language['capture_video']				=	setLanguageValue('capture_video','Capture Video',$lang,'core','default');
$language['leave_chatroom']				=	setLanguageValue('leave_chatroom','Leave Group',$lang,'core','default');
$language['invite_users']				=	setLanguageValue('invite_users','Add Members',$lang,'core','default');
$language['create_chatroom']			=	setLanguageValue('create_chatroom','Create Group',$lang,'core','default');
$language['options']					=	setLanguageValue('options','Options',$lang,'core','default');
$language['close_tab']					=	setLanguageValue('close_tab','Close',$lang,'core','default');
$language['minimize_tab']				=	setLanguageValue('minimize_tab','Minimize Tab',$lang,'core','default');
$language['close_popup']				=	setLanguageValue('close_popup','Close Popup',$lang,'core','default');
$language['login_options']				=	setLanguageValue('login_options','Login Options',$lang,'core','default');
$language['minimize']					=	setLanguageValue('minimize','Minimize',$lang,'core','default');
$language['login_mode']					=	setLanguageValue('login_mode','Select login mode',$lang,'core','default');
$language['logout']						=	setLanguageValue('logout','Logout',$lang,'core','default');
$language['logout_unsuccessful']		=	setLanguageValue('logout_unsuccessful','Logout was unsuccessful. Please try again.',$lang,'core','default');
$language['login_here']					=	setLanguageValue('login_here','Login here',$lang,'core','default');
$language['load_earlier_messages']		=	setLanguageValue('load_earlier_messages','Load Earlier Messages',$lang,'core','default');
$language['no_more_messages']			=	setLanguageValue('no_more_messages','No more messages',$lang,'core','default');
$language['type_message']				=	setLanguageValue('type_message','Type your message',$lang,'core','default');
$language['active_chats']				=	setLanguageValue('active_chats','Active Chats',$lang,'core','default');
$language['all_users']					=	setLanguageValue('all_users','All Users',$lang,'core','default');
$language['unread']						=	setLanguageValue('unread','Unread',$lang,'core','default');
$language['hello']						=	setLanguageValue('hello','Hello',$lang,'core','default');
$language['select_a_user']				=	setLanguageValue('select_a_user','To get started, select a user from the left tab.',$lang,'core','default');
$language['select_a_group']				=	setLanguageValue('select_a_group','To get started, select a group from the left tab.',$lang,'core','default');
$language['select_a_user_or_group']				=	setLanguageValue('select_a_user_or_group','To get started, select a user or group from the left tab.',$lang,'core','default');
$language['exclamation']				=	setLanguageValue('exclamation','!',$lang,'core','default');
$language['go_online']					=	setLanguageValue('go_online','You are Offline. Click here to go Online.',$lang,'core','default');
$language['chat_with_users']			=	setLanguageValue('chat_with_users','To chat with site users:',$lang,'core','default');
$language['contact_admin']				=	setLanguageValue('contact_admin','Please contact your site administrator.',$lang,'core','default');
$language['plugins']					=	setLanguageValue('plugins','Plugins',$lang,'core','default');
$language['instant_pm']					=	setLanguageValue('instant_pm','Instant PM',$lang,'core','default');
$language['edit']						=	setLanguageValue('edit','Edit',$lang,'core','default');
$language['user_currently_offline']		=	setLanguageValue('user_currently_offline','This user is currently offline',$lang,'core','default');
$language['blocked_user_message']		=	setLanguageValue('blocked_user_message','You won&#39);t recieve messages from this user.',$lang,'core','default');
$language['do_not_disturb']				=	setLanguageValue('do_not_disturb','Do not show this again',$lang,'core','default');
$language['cancel']						=	setLanguageValue('cancel','Cancel',$lang,'core','default');
$language['unblock_user']				=	setLanguageValue('unblock_user','Are you sure you want to unblock this user?',$lang,'core','default');
$language['ok']							=	setLanguageValue('ok','Ok',$lang,'core','default');
$language['delete_messages']			=	setLanguageValue('delete_messages','This PM and chat history will be deleted forever. Are you sure?',$lang,'core','default');
$language['blocked']					=	setLanguageValue('blocked','Blocked',$lang,'core','default');
$language['no_conversations']			=	setLanguageValue('no_conversations','You have no conversations. To start a conversation, click on the instant PM button in forum thread.',$lang,'core','default');
$language['logout_message']				=	setLanguageValue('logout_message','You have been logged out because your session has expired.',$lang,'core','default');
$language['last_seen']					=	setLanguageValue('last_seen','Last Seen',$lang,'core','default');
$language['last_seen_at']				=	setLanguageValue('last_seen_at','Last seen at',$lang,'core','default');
$language['n_supported_on_ios']			=	setLanguageValue('n_supported_on_ios','This feature is not supported on iOS devices.',$lang,'core','default');
$language['click_here']					=	setLanguageValue('click_here','Click here',$lang,'core','default');
$language['to_use_chat']				=	setLanguageValue('to_use_chat',' to use the chat',$lang,'core','default');
$language['disable_message_receipt']	=	setLanguageValue('disable_message_receipt','Disable Read Receipts',$lang,'core','default');
$language['use_mobileapp']				=	setLanguageValue('open_mobileapp','Use Mobile App?',$lang,'core','default');
$language['jan']						=	setLanguageValue('jan','January',$lang,'core','default');
$language['feb']						=	setLanguageValue('feb','February',$lang,'core','default');
$language['mar']						=	setLanguageValue('mar','March',$lang,'core','default');
$language['apr']						=	setLanguageValue('apr','April',$lang,'core','default');
$language['may']						=	setLanguageValue('may','May',$lang,'core','default');
$language['jun']						=	setLanguageValue('jun','June',$lang,'core','default');
$language['jul']						=	setLanguageValue('jul','July',$lang,'core','default');
$language['aug']						=	setLanguageValue('aug','August',$lang,'core','default');
$language['sep']						=	setLanguageValue('sep','September',$lang,'core','default');
$language['oct']						=	setLanguageValue('oct','October',$lang,'core','default');
$language['nov']						=	setLanguageValue('nov','November',$lang,'core','default');
$language['dec']						=	setLanguageValue('dec','December',$lang,'core','default');
$language['today']						=	setLanguageValue('today','Today',$lang,'core','default');
$language['yesterday']					=	setLanguageValue('yesterday','Yesterday',$lang,'core','default');
$language['recent_chats']				=	setLanguageValue('recent_chats','Recent',$lang,'core','default');
$language['groups']						=	setLanguageValue('groups','Groups',$lang,'core','default');
$language['more']						=	setLanguageValue('more','More',$lang,'core','default');
$language['available']					=	setLanguageValue('available','Available',$lang,'core','default');
$language['away']						=	setLanguageValue('away','Away',$lang,'core','default');
$language['busy']						=	setLanguageValue('busy','Busy',$lang,'core','default');
$language['invisible']					=	setLanguageValue('invisible','Invisible',$lang,'core','default');
$language['notifications']				=	setLanguageValue('notifications','Notifications',$lang,'core','default');
$language['blocked_users']				=	setLanguageValue('blocked_users','Blocked Users',$lang,'core','default');
$language['manage_blocked_users']				=	setLanguageValue('manage_blocked_users','Manage Blocked Users',$lang,'core','default');
$language['no_recent_chats']			=	setLanguageValue('no_recent_chats','No recent chats available',$lang,'core','default');
$language['no_groups']					=	setLanguageValue('no_groups','No groups available',$lang,'core','default');
$language['announcements']				=	setLanguageValue('announcements','Announcements',$lang,'core','default');
$language['you_are_offline']			=	setLanguageValue('you_are_offline','You\'re Offline' ,$lang,'core','default');
$language['check_internet']				=	setLanguageValue('check_internet','No internet connection','No inter' ,$lang,'core','default');
$language['chats']						=	setLanguageValue('chats','Chats',$lang,'core','default');
$language['no_group_found']				=	setLanguageValue('no_group_found','No group found',$lang,'core','default');
$language['close_user_search']			=	setLanguageValue('close_user_search','Clear search',$lang,'core','default');
$language['new_group']					=	setLanguageValue('new_group','New Group',$lang,'core','default');
$language['new_broadcast']				=	setLanguageValue('new_broadcast','New Broadcast',$lang,'core','default');
$language['broadcast_message']			=	setLanguageValue('broadcast_message','Broadcast Message',$lang,'core','default');
$language['save_changes']				=	setLanguageValue('save_changes','Save Changes',$lang,'core','default');
$language['contacts']					=	setLanguageValue('contacts','Contacts',$lang,'core','default');
$language['announcements']				=	setLanguageValue('announcements','Announcements',$lang,'core','default');
$language['view_profile']				=	setLanguageValue('view_profile','View Profile',$lang,'core','default');
$language['close_chatbox']				=	setLanguageValue('close_chatbox','Closing the chatbox will close all active plugins. Do you really want to close? ',$lang,'core','default');
$language['log_in_using']				=	setLanguageValue('log_in_using','Log in using',$lang,'core','default');
$language['bots']						=	setLanguageValue('bots','Bots',$lang,'core','default');
$language['no_bots']					=	setLanguageValue('no_bots','No bots available',$lang,'core','default');
$language['bot_info']					=	setLanguageValue('bot_info','Bot Information',$lang,'core','default');
$language['no_bot_found']				=	setLanguageValue('no_bot_found','No bot found',$lang,'core','default');
$language['sticker']					=	setLanguageValue('sticker','Sticker',$lang,'core','default');
$language['show_read_receipt']			=	setLanguageValue('show_read_receipt','Show Read Receipt',$lang,'core','default');
$language['handwrite']					=	setLanguageValue('handwrite','Handwrite Image',$lang,'core','default');
$language['audiofile']					=	setLanguageValue('audiofile','Audio File',$lang,'core','default');
$language['file']						=	setLanguageValue('file','File',$lang,'core','default');
$language['videobroadcast']				=	setLanguageValue('videobroadcast','Video Broadcast Request',$lang,'core','default');
$language['screenshare']				=	setLanguageValue('screenshare','Screenshare Request',$lang,'core','default');
$language['whiteboard']					=	setLanguageValue('whiteboard','Whiteboard Request',$lang,'core','default');
$language['writeboard']					=	setLanguageValue('writeboard','Writeboard Request',$lang,'core','default');
$language['avchat']						=	setLanguageValue('avchat','Video Call',$lang,'core','default');
$language['audiochat']					=	setLanguageValue('audiochat','Audio Call',$lang,'core','default');
$language['image']						=	setLanguageValue('image','Image',$lang,'core','default');
$language['video']						=	setLanguageValue('video','Video',$lang,'core','default');
$language['chatroom_join']				=	setLanguageValue('chatroom_join','Group Join Request',$lang,'core','default');
$language['bot_responce']				=	setLanguageValue('bot_responce','Bot Responce',$lang,'core','default');
$language['no_chats_found']				=	setLanguageValue('no_chats_found','No chats found',$lang,'core','default');
$language['membership_msg']				=	setLanguageValue('membership_msg','Sorry, you do not have access to this feature.',$lang,'core','default');
$language['access_group_guest']			=	setLanguageValue('access_group_guest','You do not have access to Group Chat.',$lang,'core','default');
$language['insufficient_credits']		=	setLanguageValue('insufficient_credits','You do not sufficient credits to use this feature.',$lang,'core','default');
$language['guest_already_exist']		=	setLanguageValue('guest_already_exist','already exists',$lang,'core','default');
$language['guest_updated_success']		=	setLanguageValue('guest_updated_success','Guest name updated successfully',$lang,'core','default');


///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

$key_mapping = array(
	'0'		=>	'chat_options',
	'1'		=>	'type_status',
	'2'		=>	'status_message',
	'3'		=>	'status_option_a',
	'4'		=>	'status_option_b',
	'5'		=>	'status_option_i',
	'6'		=>	'blank1',
	'7'		=>	'blank2',
	'8'		=>	'loggedout_tooltip',
	'9'		=>	'chat_now_tab',
	'10'	=>	'me',
	'11'	=>	'status_option_o',
	'12'	=>	'who_is_online',
	'13'	=>	'sound_notifications',
	'14'	=>	'no_friends',
	'15'	=>	'new_messages_notify',
	'16'	=>	'blank3',
	'17'	=>	'user_tab_o',
	'18'	=>	'find_a_user',
	'19'	=>	'user_online',
	'20'	=>	'user_offline',
	'21'	=>	'user_busy',
	'22'	=>	'set_my_status',
	'23'	=>	'online_status',
	'24'	=>	'popup_notifications',
	'25'	=>	'blank4',
	'26'	=>	'start_chatting',
	'27'	=>	'close',
	'28'	=>	'no_users_online',
	'29'	=>	'no_friends_online',
	'30'	=>	'status_available',
	'31'	=>	'status_busy',
	'32'	=>	'status_offline',
	'33'	=>	'status_invisible',
	'34'	=>	'status_away',
	'35'	=>	'separator',
	'36'	=>	'and',
	'37'	=>	'more_notifications',
	'38'	=>	'close_existing_popup',
	'39'	=>	'navigate_page',
	'40'	=>	'others',
	'41'	=>	'loading',
	'42'	=>	'blank5',
	'43'	=>	'my_name',
	'44'	=>	'set_my_name',
	'45'	=>	'type_name_status',
	'46'	=>	'messages_from',
	'47'	=>	'view',
	'48'	=>	'new_message_from',
	'49'	=>	'updated',
	'50'	=>	'failed',
	'51'	=>	'chat_now',
	'52'	=>	'read_message',
	'53'	=>	'spam',
	'54'	=>	'new_messages',
	'55'	=>	'send_a_message',
	'56'	=>	'loggedout',
	'57'	=>	'no_change',
	'58'	=>	'no_user_found',
	'59'	=>	'message_with_smiley',
	'60'	=>	'web',
	'61'	=>	'mobile',
	'62'	=>	'minimize_user_tab',
	'63'	=>	'minimize_setting_tab',
	'64'	=>	'clear_conversation',
	'65'	=>	'report_conversation',
	'66'	=>	'share_image',
	'67'	=>	'capture_photo',
	'68'	=>	'share_video',
	'69'	=>	'capture_video',
	'70'	=>	'leave_chatroom',
	'71'	=>	'invite_users',
	'72'	=>	'create_chatroom',
	'73'	=>	'options',
	'74'	=>	'close_tab',
	'75'	=>	'minimize_tab',
	'76'	=>	'close_popup',
	'77'	=>	'login_options',
	'78'	=>	'minimize',
	'79'	=>	'login_mode',
	'80'	=>	'logout',
	'81'	=>	'logout_unsuccessful',
	'82'	=>	'login_here',
	'83'	=>	'load_earlier_messages',
	'84'	=>	'no_more_messages',
	'85'	=>	'type_message',
	'86'	=>	'active_chats',
	'87'	=>	'all_users',
	'88'	=>	'unread',
	'89'	=>	'hello',
	'90'	=>	'select_a_user',
	'91'	=>	'exclamation',
	'92'	=>	'go_online',
	'93'	=>	'chat_with_users',
	'94'	=>	'contact_admin',
	'95'	=>	'plugins',
	'96'	=>	'instant_pm',
	'97'	=>	'edit',
	'98'	=>	'user_currently_offline',
	'99'	=>	'blocked_user_message',
	'100'	=>	'do_not_disturb',
	'101'	=>	'cancel',
	'102'	=>	'unblock_user',
	'103'	=>	'ok',
	'104'	=>	'delete_messages',
	'105'	=>	'blocked',
	'106'	=>	'no_conversations',
	'107'	=>	'logout_message',
	'108'	=>	'last_seen',
	'109'	=>	'last_seen_at',
	'110'	=>	'n_supported_on_ios',
	'111'	=>	'click_here',
	'112'	=>	'to_use_chat',
	'113'   =>  'new_broadcast',
	'114'   =>  'no_group_found',
	'115'   =>  'close_user_search',
	'116'   => 'save_changes',
	'117'   => 'broadcast_message',
	'118'   =>  'no_bot_found'
);

$language = mapLanguageKeys($language,$key_mapping,'core','default');
